chain/jsonrpc-primitives/src/types/transactions.rs

```diff
@@ -20,10 +20,14 @@ pub enum TransactionInfo {
     },
 }
 
-#[derive(thiserror::Error, Debug)]
+#[derive(thiserror::Error, Debug, Serialize)]
+#[serde(tag = "name", content = "info", rename_all = "SCREAMING_SNAKE_CASE")]
 pub enum RpcTransactionError {
     #[error("An error happened during transaction execution: {context:?}")]
-    InvalidTransaction { context: near_primitives::errors::InvalidTxError },
+    InvalidTransaction {
+        #[serde(skip)]
```

Comment:
Any reason why we currently `skip` this field? I'm looking to implement `Deserialize` for this struct.

Reply:
Resolved with https://github.com/near/nearcore/pull/4770/files#diff-b0dd5d6f8f4d2a7b135860666232d02f02a60ee1dd9d7fd65ee6afe79550e98eR30